packages/header-check/header-check.0.1.0/opam

```diff
@@ -0,0 +1,50 @@
+opam-version: "2.0"
+license: "LGPL-2.1-only WITH OCaml-LGPL-linking-exception"
+synopsis: "A tool to check and update source headers, using checksums"
+description: """\
+The header-check tool scans a source project, lists all existing headers and
+can be used to update them. It uses checksums to create identifiers for 
+headers and works with OCaml, C/C++ and Shell kinds of comments.
+"""
+authors: ["Fabrice Le Fessant <fabrice.le_fessant@ocamlpro.com>"]
+maintainer: ["Fabrice Le Fessant <fabrice.le_fessant@ocamlpro.com>"]
+homepage: "https://ocamlpro.github.io/header-check"
```

Comment:
I have a 404 on this link. Is this expected for now? Is there a different URL to add?

Same remark on the doc link below.

This is not necessarily blocking, but if you want to fix it, you might as well do it before the pull.

Reply:
Thanks for noticing, I decided to push a website there, so it should be ok for now.

Reply:
It's working now